bedrock/exp/urls.py

```diff
@@ -15,8 +15,7 @@
 urlpatterns = (
     page('opt-out', 'exp/opt-out.html'),
     page('firefox', 'exp/firefox/index.html', active_locales=['en-US', 'en-GB', 'en-CA', 'de']),
-    page('firefox/new', 'exp/firefox/new/download.html', active_locales=['en-US', 'en-GB', 'en-CA', 'de']),
-    page('firefox/new/nav', 'exp/firefox/new/download-nav.html'),
```

Comment:
Should we redirect this old URL instead of let it 404?

Reply:
Feels like unnecessary tech debt to me. It's not indexed and I don't know why anyone would have it bookmarked. No one has visited the page since the experiment ended.

Reply:
I see a small amount of daily traffic to this URL in GA, but it's pretty small. Your call 👍

Reply:
Oh I had my search parameters wrong when I did my search.

But this is a real problem. We need to figure out how people are getting here.

Reply:
I'll redirect these people to the real /new page.

But I also want to fix the root of the problem.